src/main/java/uk/gov/hmcts/reform/divorce/casemaintenanceservice/util/CaseDataUtil.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.divorce.casemaintenanceservice.util;
+
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Component
+public class CaseDataUtil {
+
+    public Map<String, Object> copyWithCaseIdRemoved(final Map<String, Object> caseData) {
```

Comment:
if it is a util I would say accept two parameters one caseData and other String `fieldToBeRemoved` rather than removing id always ? and can rename the method `getCaseDataCopyWithRemovingField(caseData,"id")` ?

Reply:
Quite right.  I have improved this util, just in case we need it somewhere else.